lib/mbr.mli

```diff
@@ -84,6 +84,9 @@ type t = private {
 }
 
 val make : Partition.t list -> (t, string) result
+
+val make : ?disk_signature:int32 -> 'a list -> 'a
```

Comment:
You got it right with the `?disk_signature:int32` part. If you can add it to the type signature of `make` above that would be great. The rest of the type function of the latter `make` is not quite right.

Reply:
Hello @reynir, thanks for the review, I just added it now.